extension/src/popup/routes.tsx

```diff
@@ -21,4 +21,8 @@ export const routes: RouteObject[] = [
     path: 'open-wallet/connect-device',
     element: <ConnectDevicePage />,
   },
+  {
+    path: 'open-wallet/ledger',
+    element: <FromLedgerWebExtension />,
```

Comment:
Does it work in browser extension?

Reply:
No, it actually does not. At least from my testing. Disabled it for now.